includes/admin/class-rop-admin.php

```diff
@@ -1003,6 +1005,76 @@ public function rop_dismiss_cron_disabled_notice() {
 
 	}
 
+	/**
+	 * Migrate the taxonomies from General Settings to Post Format for Pro users.
+	 * @since 8.5.4
+	 */
+	public function migrate_taxonomies_to_post_format() {
+
+		$global_settings = new Rop_Global_Settings();
+		// If any type of Pro is installed and active.
+		if ( $global_settings->license_type() > 0 ) {
```

Comment:
We can optimize this a little by simply returning early:

```
if ( $global_settings->license_type() < 1 ) {
return;
}
```

Reply:
I can't see the difference, if the condition from the `if` statement is not met, PHP skips the whole block, there's no performance issue there.